.github/workflows/webserver-deploy.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and deploy webserver
+
+on:
+  push:
+    paths:
+      - 'webserver/**'
+      - 'build/webserver/**'
+      - '.github/**'
+
+env:
+  EC2_HOST: "35.180.67.147"
+  DOCKER_NETWORK_BRIDGE_NAME: "totocorp-network"
+  NODE_PORT: 3001
+  RETRIES_COUNT: 5
```

Comment:
Part of this could probably be put as secrets or configuration. This seems very similar between what is done in the [user-service](https://github.com/Knoblauchpilze/user-service/blob/master/.github/workflows/deploy.yml#L14-L17).